Resources/Prototypes/_NF/Shipyard/spectre.yml

```diff
@@ -1,15 +1,25 @@
+# Author Info
+# GitHub: Wolfhauler
+# Discord: wolfhauler
+
+# Maintainer Info
+# GitHub: ThatOneGoblin25 
+# Discord: thedaarkgoblin
+
+# Shuttle Notes:
+#
 - type: vessel
   id: Spectre
-  name: NSS Spectre
-  description: A large research mothership designed to be flown nexto a small fleet of other ships including salvage and food services.
-  price: 195000
+  name: NR Spectre
+  description: A large, dashing, but dated vessel with a pure focus on research and development. It is capable of generating anomalies.
+  price: 185000
   category: Large
   group: Civilian
   shuttlePath: /Maps/_NF/Shuttles/spectre.yml
 
 - type: gameMap
   id: Spectre
-  mapName: 'NSSSpectre'
+  mapName: 'Spectre'
```

Comment:
Please use the same map name format as all other ships

Reply:
Done